lib/Conversion/TCFToLinalg/TCFToLinalg.cpp

```diff
@@ -32,6 +32,16 @@ static SmallVector<Value, 6> bypassResultShapes(Operation *op,
         op->getLoc(), ValueRange({lhsRows, rhsCols}));
     return {shape};
   }
+  // TODO: This only supports the NCHW data format. Consider other formats and lower ranks.
+  if (auto conv2dNCHW = dyn_cast<tcf::ConvNCHWOp>(op)) {
```

Comment:
This is not correct. The general formula is here: https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html

![image](https://user-images.githubusercontent.com/24583347/101712373-3e2c1780-3a4a-11eb-885e-09c7a24204af.png)

In this case, padding=0, dilation=1, kernel_size=KW or KH appropriately, and stride=1.

Reply:
OK